static/app/views/settings/projectSecurityAndPrivacy/index.tsx

```diff
@@ -70,7 +75,7 @@ export default function ProjectSecurityAndPrivacy({organization, project}: Props
         }
         endpoint={endpoint}
         relayPiiConfig={relayPiiConfig}
-        disabled={!organization.access.includes('project:write')}
+        disabled={hasAccess}
```

Comment:
```suggestion
        disabled={!hasAccess}
```
? it passed CI though